src/olympia/amo/templates/amo/category-arrow.html

```diff
@@ -5,6 +5,19 @@
   .html-rtl #{{ prefix }}-{{ key }} {
     background: url({{ STATIC_URL }}img/zamboni/arrow-left-blue.png) no-repeat right center;
   }
+  .restyle #{{ prefix }}-{{ key }},
```

Comment:
Sorry about this, but it's already pretty weird and doing this a better way was... a lot more work :disappointed: